### ISSUE TITLE
On windows Tensorflow 2.5 it gives error

### ISSUE THREAD

**Issue by mgezer** [Rishit-dagli/Gradient-Centralization-TensorFlow#23]
On windows 10 with miniconda enviroment tensorflow 2.5 gives error on centralized_gradients.py file.

the solution is change 
import keras.backend as K
with 
import tensorflow.keras.backend as K

**Comment by Rishit-dagli** [comment]
This is interesting, could you share the error?

**Comment by mgezer** [comment]
![image](https://user-images.githubusercontent.com/16152734/122589038-7733ed00-d068-11eb-9f10-0d186c086a03.png)

**Comment by Rishit-dagli** [comment]
Hi @mgezer ,
Thanks for sharing but I am unable to reproduce the error, could you probably share a Colab notebook?

**Comment by mgezer** [comment]
Hello, First of all, let me explain how it occurs on a system; by the way, I tested it not only on windows but also on macOS machine.

First of all, this is the system I set up with miniconda:
conda create -n tf25 python=3.9
conda activate tf25   
conda install jupyter notebook jupyterlab scikit-learn pandas seaborn matplotlib
pip install tensorflow==2.5 tensorflow-addons
pip install gradient-centralization-tf

within this environment, I had every time the error when I import gctf 


**_AttributeError: module 'tensorflow.python.keras.utils.generic_utils' has no attribute 'populate_dict_with_module_objects'_**


I have experimented with Google Colab, but I haven't got this error because Keras is installed separately.


I have made a little solution with a change in the file **_centralized_gradients.py_**

the line that have 
**_import keras.backend as K_**
with
**_import tensorflow.keras.backend as K_**

**Comment by Rishit-dagli** [comment]
Got it, ideally, this should soon be fixed since it is a version mismatch (https://github.com/keras-team/keras/issues/14632) and you should easily be able to solve this by downgrading TensorFlow from 2.5.x. But thanks so much for pointing this out, would you like to contribute to it?
